config/config.go

```diff
@@ -5,11 +5,11 @@ import (
 	"encoding/json"
 	"fmt"
 	"os"
-	"otc-auth/src/common"
+	"otc-auth/common"
 	"time"
 )
 
-var otcConfigPath = getHomeFolder() + "/.otc-auth-config"
+var otcConfigPath = GetHomeFolder() + "/.otc-auth-config"
```

Comment:
Might be better to use `path.Join` instead. Then Go will take care of the separator for windows and linux.